sdk/script/management-sdk.ts

```diff
@@ -67,7 +71,7 @@ class AccountManager {
 
     public isAuthenticated(): Promise<boolean> {
         return Promise<boolean>((resolve, reject, notify) => {
-            var request: superagent.Request<any> = superagent.get(this._serverUrl + urlEncode `/authenticated`);
+            var request: superagent.Request<any> = (<any>superagent.get(this._serverUrl + urlEncode `/authenticated`)).proxy(proxy);
```

Comment:
Just to make sure we run cleanly if the proxy is not set, could we change these three instances to have this on another line:

`if (proxy) (<any>request).proxy(proxy);`